topics/assembly/tutorials/debruijn-graph-assembly/tutorial.md

```diff
@@ -18,7 +18,7 @@ SPAdes is a de novo genome assembler written by Pavel Pevzner's group in St. Pet
 > In this tutorial, we will deal with:
 >
 > 1. [Get the data](#get-the-data)
```

Comment:
ah yes, good point ..will add that to my PR with fixes for the broken links reported by htmlproofer :)